src/spacehauc-i2c-dev.cpp

```diff
@@ -352,15 +342,7 @@ PWMcontroller::PWMcontroller(int file, uint8_t address, uint8_t ID_register,
  * @return Returns true/false depending on success/failure.
  */
 bool PWMcontroller::initRGB_PWMcontroller() {
-  if (!setFreq(400)) {
-    return false;
-  }
-  // invert direction of current flow
```

Comment:
Would it be ok to leave these comments? They were left here so that if we later need to know how to invert the direction of current flow, we won't have to research it agian